rules/0.17-allow-new-players.py

```diff
@@ -14,11 +15,12 @@ def should_allow(pr):
 
     (points_user, points_name, points_change) = bonuses[0]
 
-    if points_user in util.users():
-        raise Exception('Cannot create an existing user')
+    if pr.author() != points_user:
+        raise Exception('New players should submit their own PRs, but %s submitted the PR to add %s' %
+                        (pr.author(), points_user))
 
     if points_name != 'initial':
-        raise Exception('New player bonus value must be called "initial"')
+        raise Exception('New player bonus value is called "%s" instead of "initial"' % points_name)
```

Comment:
What I'm not clear on is how we know we're looking at a PR for a new user at this point? If an existing user tries to add a new, non-`'initial'` bonus file won't this exception still throw?

Reply:
Yes. And that's fine, because this is an allow rule. If it throws, we just move to the next rule. The general pattern for the allow rules is that we assume the PR in question is trying to pass that rule, and throw exceptions with wording to match.

Reply:
Doesn't a throw still look like a failure in CI?

Reply:
`validate.py` traps the exceptions and prints them.

Reply:
Why does CI think https://travis-ci.com/jeffkaufman/nomic/builds/103835754 failed then? Is the goal not to have everything passing before merging?